### PR TITLE
Check for short messages in accordance with OpCodes Spec para 2.1.1

### PR DESCRIPTION
**Pull request by MartinDaCosta53**

### CODE REVIEW

File: src/MinimumNodeService.cpp [hunk 1]
```diff
@@ -212,18 +212,25 @@ Processed MinimumNodeService::handleMessage(unsigned int opc, CANFrame *msg)
 
       if (nn == module_config->nodeNum)
       {
-        byte paran = msg->data[3];
+        if (msg->len < 4)
+        {
+          controller->sendGRSP(OPC_RQNPN, getServiceID(), CMDERR_INV_CMD);
```
Comment (SvenRosvall) [inline src/MinimumNodeService.cpp:217]: Should this condition also send a CMDERR?

Comment (SvenRosvall) [reply src/MinimumNodeService.cpp:217]: Sorry, I see now that this is a VLCB specific check. So no CMDERR.

Comment (MartinDaCosta53) [reply src/MinimumNodeService.cpp:217]: It doesn't seem to be specified in CBUS and Duncan didn't in his code, so I think not.

Comment (MartinDaCosta53) [reply src/MinimumNodeService.cpp:217]: Ian doesn't in his code, either

**Pull request merged**